ls1intum/Artemis
# `Lectures`: Polish design and usability of lecture unit management view

### Checklist
#### General


#### Client

### Motivation and Context



The exercise management view currently looks weird and totally out of line with how the rest of Artemis looks, especially with those rounded buttons and large border radius boxes. Additionally, no drag and drop sort is possible.

![grafik](https://user-images.githubusercontent.com/23171488/183939348-42f26ac5-2878-4839-a12e-73d9ac1818e2.png)

### Description


- Changed the design of this page to look more like all other components we have.
- Grouped all available buttons on the right instead of splitting them between left and right
- Enabled drag and drop sorting
- Moved the "Create new" buttons to the top right as that's like it is everywhere else

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Lecture

1. Open the lecture unit management view of the lecture
2. Add a few units of all types
3. Reorder them with drag and drop and make sure that order is persisted (aka, still there after a reload)
4. Make sure that all buttons still work
5. Make sure that you can still expand them and the content works

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots




https://user-images.githubusercontent.com/23171488/183942547-86d47d67-3956-453e-afb6-7fc4ad72064c.mov
